src/main/java/echoserver/Client.java

```diff
@@ -0,0 +1,7 @@
+package echoserver;
+
+import java.io.IOException;
+
+public interface Client {
```

Comment:
What is a client?

Reply:
I included this interface for the purpose of creating FakeEchoClients

Reply:
Maybe this is one I can rename to Echoable?

Reply:
Yeah, I think I would just suggest against using language like "client" because that can lead to confusion in your server implementation.